src/constants.ts

```diff
@@ -52,3 +52,5 @@ export const DEFAULT_CACHING_SAFE_LAG = 60 * 60 * 24; // 1 Day
 
 export const UBA_BOUNDS_RANGE_MAX = BigNumber.from(String(Number.MAX_SAFE_INTEGER)).mul(utils.parseEther("1.0"));
 export const UBA_BOUNDS_RANGE_MIN = UBA_BOUNDS_RANGE_MAX.mul(-1);
+
+export const RISK_LABS_RELAYER_ADDRESS = "0x428ab2ba90eba0a4be7af34c9ac451ab061ac010";
```

Comment:
nit: should this be checksummed? I also don't like this name
```suggestion
export const DEFAULT_SIMULATED_RELAYER_ADDRESS = "0x428ab2ba90eba0a4be7af34c9ac451ab061ac010";
```

Reply:
We could checksum it, but it shouldn't affect the usage in our logic because ethers internally checksum the addresses it receives. For readability, I've checksum the address & refactored the naming. https://github.com/across-protocol/sdk-v2/pull/427/commits/2b6a569af4bbd07119bde8c56dce11bd58c8a18b

Reply:
I just think we should always store hardcoded addresses as checksummmed as it can only help